### PR TITLE
Add unit tests to knativeserving_lifecycle

### PR DESCRIPTION
**Pull request by houshengbo**
Fixes #

## Proposed Changes

*
*
*

**Release Note**

```release-note

```

### CODE REVIEW

File: pkg/apis/serving/v1alpha1/knativeserving_lifecycle.go [hunk 1]
```diff
@@ -20,7 +20,14 @@ import (
 	"knative.dev/pkg/apis"
 )
 
+const (
+	// KnativeServingConditionReady is set when the KnativeServing's latest
+	// underlying revision has reported readiness.
+	KnativeServingConditionReady = apis.ConditionReady
```
Comment (jcrossley3) [inline pkg/apis/serving/v1alpha1/knativeserving_lifecycle.go]: What problem is this solving?

**Review by jcrossley3 — COMMENTED** [review]
Apologies if I'm missing something obvious, but aren't these already covered by the upstream `condition_*_test.go` tests? https://github.com/knative/pkg/tree/master/apis

**Comment by houshengbo** [comment]
/test pull-knative-serving-operator-integration-tests-on-latest-serving
There is an additional job added. It can be triggered manually.

**Comment by aliok** [comment]
@houshengbo is it ok to close this PR? looks pretty old.
getting notifications like "PR needs rebase" otherwise.

**Comment by houshengbo** [comment]
Close this PR, because this repo is deprecated.

**Pull request closed without merging**